void main() {
//  throw new FormatException('格式化错误！');
//throw new NullThrownError();

//  throw '这是一个异常！！';

//  try {
//    throw new NullThrownError();
//  } on Error {
//    print('error 类型异常被捕获');
//  } on OutOfMemoryError {
//    print('error 类型异常被捕获');
//  } on Exception catch(e) {
//    print('error 类型异常被捕获');
//  } on Error {
//  } catch (e, s) {
//    //e:异常对象，s:StackTrace对象
//    print(e);
//    print(s);
//  }






}
